backend/src/routes/v0/llm/__init__.py

```
@@ -5,8 +5,9 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from langchain.chat_models import init_chat_model
 
-from src.models import ProtectedUser, User
-from src.entities import Answer, ChatInput, NewThread, ExistingThread
+from src.constants.examples import NEW_THREAD_API_TOOLS, NEW_THREAD_QUERY_EXAMPLE
```

Ruff (0.11.9)

`src.constants.examples.NEW_THREAD_API_TOOLS` imported but unused

(F401)